optimum/intel/openvino/configuration.py

```diff
@@ -473,8 +501,13 @@ def __init__(
         self.compression = kwargs.get(
             "compression", None
         )  # A field for backward-compatability of training-time compression parameters
-        bits = self.quantization_config.bits if self.quantization_config else None
-        self.dtype = "int" + str(bits) if isinstance(bits, int) else dtype
+        if self.quantization_config is not None:
+            if isinstance(self.quantization_config, OVWeightQuantizationConfig):
+                self.dtype = self.quantization_config.weight_format
+            else:
+                self.dtype = "int8"
```

Comment:
should it be :
```suggestion
                self.dtype = "int" + str(bits) if isinstance(bits, int)
```

Reply:
For activation quantization currently only 8-bits option is supported. But using `bits` explicitly is fine too.

Reply:
OK I missed this part was for dynamic quantization only, then my bad your first suggestion was actually the most adapted. Could make sense to introduce `w_bit` / `a_bit` values in the future to clarify things when saving the configuration, as for dynamic quantization the weights could be quantized to 4-bit with `OVDynamicQuantizationConfig(bits=4, ...)` in which case `w_bit=4` and `a_bit=8`